src/components/TodoForm/TodoForm.jsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { userType } from '../Types/types';
+import './TodoForm.scss';
+
+export class TodoForm extends React.Component {
+  state = {
+    title: '',
+    name: '',
+    noTitle: false,
+    noName: false,
+  };
+
+  inputTitleField = (e) => {
+    this.setState({
+      title: e.target.value,
+    });
+  };
+
+  selectNameField = (e) => {
+    this.setState({
+      name: e.target.value,
+    });
+  };
+
+  addNewTodo = (e) => {
+    e.preventDefault();
+    const { title, name, noTitle, noName } = this.state;
+
+    if (!title || !name) {
+      this.setState({
+        noTitle: !title,
+        noName: !name,
+      });
```

Comment:
if you want convert a primitive value (string) into a boolean you need !! operator